src/dbnode/storage/shard_race_prop_test.go

```diff
@@ -127,7 +133,29 @@ func anyIDs() gopter.Gen {
 }
 
 func TestShardTickWriteRace(t *testing.T) {
-	shard, opts := propTestDatabaseShard(t, 10)
+	parameters := gopter.DefaultTestParameters()
+	seed := time.Now().UnixNano()
+	parameters.MinSuccessfulTests = 100
+	parameters.MaxSize = 10
+	parameters.Rng = rand.New(rand.NewSource(seed))
+	properties := gopter.NewProperties(parameters)
+
+	properties.Property("Concurrent Tick and Write doesn't deadlock", prop.ForAll(
+		func(tickBatchSize int) bool {
+			testShardTickWriteRace(t, int(tickBatchSize))
+			return true
+		},
+		gen.IntRange(1, 10).WithLabel("tickBatchSize"),
```

Comment:
also add a parameter for number of ids to `testShardTickWriteRace` and make both tickBatchSize and numberOfIds range from (1,100)